css/styleguide.scss

```diff
@@ -10,14 +10,95 @@ body {
   height: auto;
 }
 
+// USWDS Overrides
+
+.usa-footer-secondary_section {
+  background-color: $color-primary-alt-lightest;
+}
+
+.usa-graphic_list .usa-media_block-img {
+  max-width: 7rem;
+}
+
 .usa-hero {
-  background-image: url('#{$site-image-path}/home/fireworks.jpg');
+  background-image: url('#{$site-image-path}/home/wds-home-new-2x.png');
+  padding-bottom: 7.2rem;
+  padding-top: 3.5rem;
 }
 
 .usa-hero-callout .usa-button {
   margin-top: 2em;
 }
 
+.usa-hero-callout {
+  border-radius: 0.3rem;
+  max-width: 46rem;
+  padding: 3.6rem 3rem;
+
+  & > *:first-child {
+    margin-bottom: 0;
+  }
+}
+
+.usa-hero-callout-alt {
+  font-size: 2.7rem;
+}
+
+.usa-media_block-body {
+  & > *:first-child {
+    font-size: $media-block-header-font-size;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  & > p {
+    margin: 0.5rem 0;
+    font-size: $media-block-body-font-size;
+  }
+}
+
+.usa-media_block-img {
+  float: none;
+  margin-bottom: 1rem;
+}
+
+.usa-section-dark p {
+  margin-top: 0.8rem;
+  font-weight: $font-light;
+}
+
+// Override for 4x1 media grid when it collapses to 2x2
+@media screen and (max-width: 1200px) and (min-width: 600px) {
```

Comment:
Interesting, there aren't any SASS variables we can reuse instead of hard-coding `1200px` and `600px` here?  Just curious, I'm not very familiar w/ the USWDS SASS/CSS yet.